Common/Header/device.h

```diff
@@ -167,7 +167,11 @@ struct DeviceDescriptor_t {
   unsigned ErrTx;
   // Com ports hearth beats, based on LKHearthBeats
   unsigned HB;
-  
+#ifdef DEVICE_SERIAL
+  int HardwareId;
+  int SerialNumber;
+  double SoftwareVer;
+#endif
```

Comment:
why store number instead of original string ?

Reply:
Just moved it from the without changing data format.